packages/node/src/configure/configure.module.ts

```diff
@@ -115,7 +115,17 @@ export class ConfigureModule {
         ),
         config.root,
       ).catch((err) => {
-        logger.error(err, 'Create Subquery project from given path failed!');
+        if (JSON.stringify(err.message).includes(RUNNER_ERROR_REGEX)) {
```

Comment:
Could this be a function in another package so we can share it between different sdks?